Automattic/jetpack
# CRM: Adding back and unminifying font-awesome.css

## Proposed changes:

* During the process of moving CRM over to the monorepo, the font-awesome.min.css file has been removed. The [new build script](https://github.com/Automattic/jetpack/pull/28578) will remove min.css files before rebuilding from CSS files, so instead of working around that one approach is to unminify the file. Another is to refer to a CDN instead of adding font-awesome files. This PR unminifies the file.

### Other information:


## Jetpack product discussion

https://github.com/Automattic/jetpack/pull/28578#issuecomment-1420559609 

## Does this pull request change what data or activity we track or use?

No.

## Testing instructions:


* Once https://github.com/Automattic/jetpack/pull/28578 is merged, then this can be tested by creating a new Jurassic Ninja site with the [Beta plugin](https://github.com/Automattic/jetpack-beta) installed and active, and under Jetpack CRM on the Beta testing dashboard add this feature branch.
* To test a build using a [local development installation](https://github.com/Automattic/jetpack/blob/trunk/docs/development-environment.md), run `jetpack build plugins/crm`, or `jetpack build --production plugins/crm` to test the build as if it were running in production.
* Previously console errors were showing while navigating CRM dashboard pages when this file was missing - these should be gone now (and the file properly minified in a production build).
